app/controllers/application_controller.rb

```diff
@@ -127,12 +128,12 @@ def after_sign_in_path_for(resource)
     stored_location_for(resource) || user_path(resource)
   end
 
-  def after_sign_out_path_for(resource)
+  def after_sign_out_path_for
```

Comment:
If `after_sign_out_path_for` is still used anywhere (I doubt it), this will cause `ArgumentError: wrong number of arguments (1 for 0)`